python/dlr/tf_model.py

```diff
@@ -0,0 +1,154 @@
+# coding: utf-8
+import logging
+import os
+import tensorflow as tf
+
+# A prefix that will be prepended to the names in graph_def
+PREFIX = "import"
+
+
+def _load_frozen_graph(frozen_graph_file, device):
+    logging.info("Loading frozen graph: {}".format(frozen_graph_file))
+    with tf.device(device):
+        with tf.gfile.GFile(frozen_graph_file, 'rb') as f:
+            graph_def = tf.GraphDef()
+            graph_def.ParseFromString(f.read())
+        with tf.Graph().as_default() as graph:
+            tf.import_graph_def(graph_def, name=PREFIX)
+        return graph
+
+
+def _get_input_and_output_names(graph):
```

Comment:
_get_input_and_output_names -> get_input_and_output_names, would be good to add description for this func

Reply:
added docs